src/provisioner/provisioner.py

```diff
@@ -282,6 +282,17 @@ def cluster_upgrade(cortx_conf_url: str, force_override: bool = False):
         Paramaters:
         [IN] CORTX Config URL
         """
+        # query to get cluster health
+        _disruptive_upgrade = os.getenv('DISRUPTIVE_UPGRADE', str(const.DISRUPTIVE_UPGRADE))
+        if _disruptive_upgrade.upper() != "TRUE":
+            _healthy, _status = CortxProvisioner._get_cluster_health()
```

Comment:
Change this to below format:
is_healthy, resource_status = CortxProvisioner.get_health_status("cluster")

Reply:
To simply, we should just have it as 
healthy = CortxProvisioner.is_cluster_healthy() underneath it can make a call to HA for health_status etc.

Reply:
@ujjwalpl I'm with this API, but guess we discussed that resource type should also be passed as an argument...

Reply:
> Those are internals of the implementation for which arg can be avoided.
> 
> ```
> CortxProvisioer: 
>     def is_cluster_healthy():
>         CortxProvisioer._get_resource_health(resource="cortx")
> 
>     def _get_resource_health(resource):
>        # Send to HA and obtain health
> ```

Agree with this...

Reply:
done